index.js

```diff
@@ -60,7 +60,7 @@ var torrentStream = function (link, opts, cb) {
   var metadata = link.infoBuffer || null
   var infoHash = link.infoHash
 
-  if (!opts) opts = {}
+  opts = Object.assign({}, opts || {})
```

Comment:
`opts || {}` can be simplified to `opts`, since `Object.assign()` accepts `null`/`undefined` as the second argument

Reply:
Done, thank you.